### ISSUE TITLE
WCAG 2.1.2: No Keyboard Trap

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#5143]
**Level:** A
**Principle:** Operable
**Result:** *pending*

### Resources
- [WCAG 2.2 Success Criterion](https://www.w3.org/WAI/WCAG22/Understanding/no-keyboard-trap.html)

### Audit Checklist
- [ ] Review previous Audit Result
- [ ] Review work identified in previous audit and list open issues.
- [ ] Check for any other relevant open issues.
- [ ] Retest open issues, are these issues still present?
- [ ] Test for any new issues that may have been introduced.
- [ ] Determine compliance status, add this to issue name.
- [ ] Add any new issues to the Accessibility project for triage.

### Previous audit
 - link previous audit and summarise what was find and the outcome.

 ### Progress since last audit

 ### Audit Notes

 ### Work arising

**Comment by StephDriver** [comment]
## Manual Testing

All sample pages passed, but there was a one issue with the article page on OLH.  Where there is a figure in the article, the figure may be enlarged.  The enlarged figure appears as a popup, which then must be dismissed to return to the article. It can only be dismissed through the ESC key, the close button is not reachable by keyboard / no keyboard focusable. This traps users to the popup, unless they know to use the ESC key.  

I would expect the majority of keyboard users to try the ESC key as a standard exit process, so this is not a true keyboard trap.  however, as a close button is displayed, it is not ideal that this is not focusable.
